code/backend/pages/03_Delete_Data.py

```diff
@@ -73,6 +73,7 @@ def load_css(file_path):
                 )
                 if len(files_to_delete) > 0:
                     st.success("Deleted files: " + str(files_to_delete))
+                    st.experimental_rerun()
```

Comment:
Does this essentially refresh the page to regenerate the list?

Reply:
Yes.